site/content/contribute/why-contribute/_index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Section 1 — Why you might contribute"
+heading: "Section 1 — Why you might contribute"
+description: ""
+date: "2022-10-01T12:00:00-00:00"
+weight: 1
+---
+
+## Subsection 1 — You’ve found a bug
+
+1. If the bug fix that you’re proposing would be larger than 20 lines of code, [create a GitHub issue](https://github.com/mattermost/mattermost-server/issues).
+    1. You can speed up the process by asking about the issue in the [Contributors](https://community.mattermost.com/core/channels/tickets) or [Developers](https://community.mattermost.com/core/channels/developers) channels.
+    2. [Here’s a good example of a ticket](https://github.com/mattermost/mattermost-mobile/pull/364/commits/7a97451b62fee4022edac4c0395ad0a5cbf1bb66) that is small enough to not need a ticket while still being incredibly helpful.
+2. If you’ve volunteered to take the ticket once its active, or if your fix is too small to warrant the ticket, fork the applicable repo and start making your changes.
+3. Create a PR on the `master` branch of the applicable repo. If there is an associated [Jira](https://mattermost.atlassian.net/issues/?jql=) or GitHub issue, your PR should begin with the issue ID (e.g. `[MM-394]` or `[GH-394]`). Our GitHub PR template will walk you through the creation of your PR.
```

Comment:
Link to GitHub PR template from last line:
"Our GitHub PR template" should ideally point to somewhere.
I know Ben worked on a template, but not sure if his templates are already available - https://docs.google.com/document/d/11uubL-_RBEML6r3yOmDRCcTyT8CAJljiKjNdr56kpDg/edit

Reply:
@azigler - Do you know if there's Ben's GitHub templates are formalized such that we could point readers somewhere specific as per Jordana's feedback?

Reply:
I'm pretty sure those templates are part of what he wants to give developers when they spin off an open source library. I wasn't involved in making them so I can't weigh in on how "formalized" they are, unfortunately.

Reply:
@jordanafung - I've noted this as a follow-up task we can take action on in a future iteration.